dev/com.ibm.ws.kernel.service/bnd.bnd

```diff
@@ -58,9 +58,10 @@ instrument.classesExcludes: com/ibm/ws/kernel/service/utils/resources/*.class, \
 	com.ibm.ws.kernel.boot.core;version=latest,\
 	com.ibm.ws.kernel.boot.nested;version=latest,\
 	com.ibm.ws.kernel.security.thread;version=latest,\
-	com.ibm.ws.logging.core;version=latest, \
-	com.ibm.ws.org.osgi.annotation.versioning;version=latest, \
-	org.eclipse.osgi;version=latest
+	com.ibm.ws.logging.core;version=latest,\
+	com.ibm.ws.org.osgi.annotation.versioning;version=latest,\
+	org.eclipse.osgi;version=latest,\
+	io.openliberty.checkpoint
```

Comment:
As Sam and I discussed, remove this dependency and throw a runtime exception instead of CheckpointFailedException